scrape.py

```diff
@@ -25,24 +29,55 @@ def main():
     companies = text_file.read().split('\n')
     text_file.close()
 
-    driver = webdriver.Chrome(executable_path=r'/home/arthur/Programming/LeetcodePremiumScraper/chromedriver', chrome_options=opts)
+    driver = webdriver.Chrome(executable_path=r'./chromedriver', chrome_options=opts)
     driver.maximize_window()
     driver.get(login_url)
 
-    driver.find_element_by_id("id_login").send_keys("")
-    driver.find_element_by_id("id_password").send_keys("")
-    driver.find_element_by_id("id_password").submit()
-    time.sleep(4)
+    username = input("Enter username/email: ")
+    passwd = input("Enter password: ")
+
+    time.sleep(5)
+
+    while True:
+        try:
+            driver.find_element_by_id("id_login").send_keys(username)
+            driver.find_element_by_id("id_password").send_keys(passwd)
+            driver.find_element_by_id("signin_btn").click();
+            break;
+        except:
+            time.sleep(1)
 
+    time.sleep(5)
 
     for company in companies:
         lc_url = "https://leetcode.com/company/" + company
         driver.get(lc_url)
-        #driver.find_element_by_class_name("reactable-th-frequency").click()
 
-        periods = ["6months", "1year", "2year", "alltime"]
+        counter = 0
+        periodNames = ['6months', '1year', '2year', 'alltime']
+        periods = ["react-select-2--option-0", "react-select-2--option-1", "react-select-2--option-2", "react-select-2--option-3" ]
         for period in periods:
```

Comment:
Can you change this to:

`for period_selector, period_name in zip(periodSelectors, periodNames):`

This should allow you to get rid of the `counter` variable